library/Vanilla/Utility/SchemaFilters.php

```diff
@@ -43,8 +43,10 @@ public static function decodeValue($value, ValidationField $field) {
             if ($value === null) {
                 try {
                     $value = unserialize($value);
+                    // @CodeCoverageIgnoreStart
```

Comment:
Why do we want to ignore coverage here? Also, I don't know if it actually matters, but this tag usually starts with a lowercase C (e.g. [\@codeCoverageIgnoreStart](https://phpunit.de/manual/3.7/en/code-coverage-analysis.html#code-coverage-analysis.ignoring-code-blocks))

Reply:
Todd said he didn't think that part of the code was hittable, so we should ignore the code coverage. I made it camel case and am pushing the change.